account_comment_template/views/report_invoice.xml

```diff
@@ -5,17 +5,24 @@
         id="report_invoice_document_comments"
         inherit_id="account.report_invoice_document"
     >
-
```

Comment:
@victoralmau just noticed the`note1` and `note2` fields were restored in `14.0`(ref. [link](https://github.com/OCA/account-invoice-reporting/pull/165/commits/4d53d68f8a6f04657878a06255a766617355b2f4#diff-2f01dbf1fee3a3f5fd1717f36eb46d4e4617de5c0130e0b14519b52457402facR25-R26)).
Considering that they were removed in `13.0` as https://github.com/OCA/account-invoice-reporting/pull/174#discussion_r625996270 I think we should go back. Could you take the occasion to remove again them in this PR? What do you kink?

Reply:
That fields are removed in this PR. Do you want them to be? There are several reasons for removing them in v13:

- Now you can add more than one template to be put in the invoices, added according sequence.
- They provoked a very large duplication of data with barely added value.
- People got confused if they modify the notes and then select again another template, losing that changes.

v14 has this because the migration was done directly from v12 to v14 without waiting for the v13 refactoring, but I think we should go the new way even in this version.

Reply:
> That fields are removed in this PR.
Ok I see, I have misread the changes, sorry for the noise.

> I think we should go the new way even in this version.

Ok, let's go ahead.